build.gradle.kts

```diff
@@ -325,6 +325,8 @@ tasks.register("python38PostCommit") {
   dependsOn(":sdks:python:test-suites:direct:py38:postCommitIT")
   dependsOn(":sdks:python:test-suites:direct:py38:hdfsIntegrationTest")
   dependsOn(":sdks:python:test-suites:portable:py38:postCommitPy38")
+  // TODO: https://github.com/apache/beam/issues/22651
```

Comment:
nit: @AnandInguva  in Beam we typically mention a TODO, and then add a short description that follows the TODO. Full description can be in the TODO itself.

For example:

// TODO(https://github.com/apache/beam/issues/22651): Build docker image for tensor RT at runtime.

Also this todo seems a bit out of place here.